base/compiler/ssair/ir.jl

```diff
@@ -304,9 +304,9 @@ end
 
 function getindex(x::IRCode, s::SSAValue)
```

Comment:
I think you forgot to add `getindex(::IRCode, ::Int)` method?
And I wonder if this refactor would make sense:
- `getindex(::IRCode, ::Int)` returns `Instruction` object
- `getindex(::IRCode, ::SSAValue)` returns whatever statement
and the same logic for `setindex!`, and ideally for `IncrementalCompact` as well.

Reply:
No, I just messed up the title.